src/components/Actions/Actions.vue

```diff
@@ -184,6 +188,13 @@ export default {
 			type: String,
 			default: 'action-item__menutoggle--default-icon',
 		},
+		/**
+		 * Aria label for the actions menu
+		 */
+		ariaLabel: {
+			type: String,
+			default: t('Actions'),
```

Comment:
Not sure you can use this here. Mixins scope doesn't go to props I think

Reply:
`import { t } from '../../l10n'`